WaveTournament/index.js

```diff
@@ -154,4 +160,26 @@ socket.onmessage = event => {
 			
 		}
 	}
+	if(!scoreVisibleTemp) {
+		if(chatLen != data.tourney.manager.chat.length) {
+			// There's new chats that haven't been updated
+			
+			if(chatLen == 0 || (chatLen > 0 && chatLen > data.tourney.manager.chat.length)) {
+				// Starts from bottom
+				chats.innerHTML = "";
+			}
+			
+			// Add the chats
+			for(var i=chatLen; i < data.tourney.manager.chat.length; i++) {
+				tempClass = data.tourney.manager.chat[i].team
+				chats.innerHTML += "<div class='chat'><div class='wholeChat'><div class='chatTime'>"+ data.tourney.manager.chat[i].time +"</div><div class='chatName "+tempClass+"'>"+ data.tourney.manager.chat[i].name +":\xa0</div><div class='chatText'>"+ data.tourney.manager.chat[i].messageBody +"</div></div></div>";
```

Comment:
Please change += <div> to more appropriate in this case document.createElement and document.append 
```js
let parent = document.createElement("div");
let p = document.createElement("p");
p.setAttribute("class", tempClass)
parent.append(p);
```

Reply:
Consider it done 👍